docs/src/algorithms/discretization.md

```diff
@@ -5,3 +5,62 @@ discretize
 DiscretizationMethod
 FIST
 ```
+# FIST  
+Examples for the FIST algorithm. 
+
+### Usage  
+  
+```@example FIST
+using Meshes
+using Plots; gr(size = (500,300))
+
+poly = PolyArea(Point2f[(0.22926679f0, 0.47329807f0), (0.45326096f0, 0.2665109f0), (0.51871038f0, 0.22148979f0), (0.3152017f0, 0.31646582f0), (0.5953296f0, 0.62900037f0), (0.5951828f0, 0.6113712f0), (0.74374446f0, 0.72971905f0), (0.29018405f0, 0.5538437f0), (0.25439468f0, 0.5678829f0), (0.22926679f0, 0.47329807f0)], [])
+mesh = discretize(poly, FIST())
+plot(plot(poly), plot(mesh))
+```
+###### Examples of polygons and their corresponding FIST-discretized triangular meshes:  
+```@eval 
+using Plots
+using Meshes
+using Markdown
```

Comment:
If I'm not wrong, this only needs to be done once; you can just define a setup block, import what you need there, and don't reimport every time.